commons/src/main/java/com/ozonehis/eip/utils/Utils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2021, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Utils {
```

Comment:
Not sure I understand the motivation in doing this. Could you explain a little @VaishSiddharth?

Reply:
I wanted to keep the shutdown method separate from the Handler logic so that we can reuse the shutdown method in future if the use case arises.
Also `Utils` class can be useful to add more utility functions.

Reply:
Happy to move the logic to `ShutdownHandler` if the team suggests so

Reply:
I haven't really looked at the code and don't want to interfere too much.

There is no wrong way. More a question of deciding if it's overkill to introduce a `Utils` class for just **one** util method, rather than keeping it embedded elsewhere ahead of a future migration to a `Utils` class _later_ when it's more worthwhile.

Let's ask a third voice, @Ruhanga?

Reply:
In this scenario, it feels a bit excessive to isolate shutdown logic in a separate Utils class. I can't think of a use case where shutting down an application wouldn't involve calling a responsible handler for it. It would be more beneficial to make `ShutdownHandler` as generic and versatile as possible to manage shutdown processes.